inference-engine/samples/common/format_reader/CMakeLists.txt

```diff
@@ -41,3 +41,7 @@ target_include_directories(${TARGET_NAME} PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}"
 
 set_target_properties(${TARGET_NAME} PROPERTIES COMPILE_PDB_NAME ${TARGET_NAME}
                                                 FOLDER cpp_samples)
+
+if(COMMAND add_clang_format_target AND NOT IE_SAMPLE_EXCLUDE_CLANG_FORMAT)
```

Comment:
`IE_SAMPLE_EXCLUDE_CLANG_FORMAT` is not available here. I think we can drop this check